### PR TITLE
Add coverity scan

### PR DESCRIPTION
**Pull request by robertsipka**

### CODE REVIEW

**Comment by robertsipka** [comment]
@daeyeon Could you please check it?

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ dist: trusty
 sudo: required
 
 env:
+  matrix:
```
Comment (LaszloLango) [inline .travis.yml]: Is this label necessary?

Comment (robertsipka) [inline .travis.yml]: Yes, because I've added the global key. It's required to take differentiate between those two cases.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 # JerryScript: JavaScript engine for the Internet of Things
 [![License](https://img.shields.io/badge/licence-Apache%202.0-brightgreen.svg?style=flat)](LICENSE)
 [![Build Status](https://travis-ci.org/jerryscript-project/jerryscript.svg?branch=master)](https://travis-ci.org/jerryscript-project/jerryscript)
+<a href="https://scan.coverity.com/projects/jerryscript-project">
```
Comment (daeyeon) [inline README.md]: For consistency, I think you could change this as follows:
``[Coverity Scan Build Status](https://scan.coverity.com/projects/jerryscript-project)`` :-)

Comment (robertsipka) [inline README.md]: @daeyeon : Thanks! I've updated it.

File: .travis.yml (hunk 1 above)
Comment (LaszloLango) [inline .travis.yml]: In that case please increase the indentation of "OPTS" lines below.

File: tools/run-tests.py [hunk 3]
```diff
@@ -323,6 +324,12 @@ def run_buildoption_test(options):
 
     return ret
 
+def run_coverity_scan(options):
```
Comment (LaszloLango) [inline tools/run-tests.py]: Why is this change needed? Why `build_command: "tools/build.py --clean"` in travis.yml at line 37 isn't enough?

Comment (robertsipka) [inline tools/run-tests.py]: That would be simplier, yes. I think about it, but I'm not so familiar with the travis config, and it's needed by the `script` command in travis.yml at line 71. Do you have any suggestion how to avoid it?

Comment (robertsipka) [inline tools/run-tests.py]: @LaszloLango : I've updated this patch based on what we discussed offline.
